src/main/kotlin/no/nav/tiltakspenger/vedtak/rivers/PersonopplysningerMottattRiver.kt

```diff
@@ -32,23 +36,31 @@ internal class PersonopplysningerMottattRiver(
 
     override fun onPacket(packet: JsonMessage, context: MessageContext) {
         LOG.info("Received personopplysninger")
-        LOG.debug { vedtakClient }
         SECURELOG.info("Received personopplysninger for ident id: ${packet["ident"].asText()}")
-//
-//        //Metrics.mottakskanalInc(packet["mottaksKanal"].asText())
-//
-//        val personopplysningerMottattHendelse = PersonopplysningerMottattHendelse(
-//            aktivitetslogg = Aktivitetslogg(),
-//            journalpostId = packet["journalpostId"].asText(),
-//            ident = packet["ident"].asText(),
-//            personopplysninger = mapPersonopplysninger(
-//                dto = packet["@løsning.personopplysninger.person"].asObject(PersonopplysningerDTO::class.java),
-//                innhentet = packet["@opprettet"].asLocalDateTime(),
-//                ident = packet["ident"].asText(),
-//            ),
-//        )
-//
-//        innsendingMediator.håndter(personopplysningerMottattHendelse)
-//        søkerMediator.håndter(personopplysningerMottattHendelse)
+
+        runCatching {
```

Comment:
Det ser ut som at du har en runCatching uten noen håndtering av feilene. Jeg er litt usikker på hva som skjer da, om de blir slukt eller noe annet. Jeg tror kanskje du enten bør legge til en .onFailure, eller fjerne runCatching.

Så savner jeg kanskje litt logging også, ala det i ArenaTiltakMottattRiver

Reply:
Oppdatert PR-en @haagenhasle. 

Her ble jeg litt usikker på hvor mye vi trenger å logge, så jeg la bare på logging av fnr-feltet til secureLogs vha. noen nye funksjoner i Extensions.ts, istedetfor hele packet'n som gjøres for ArenaTiltakMottattRiver... Hva tenker du om det, trenger vi noe mer?